mne/viz/topomap.py

```diff
@@ -915,9 +950,11 @@ def _plot_topomap(data, pos, vmin=None, vmax=None, cmap=None, sensors=True,
     patch_ = _get_patch(outlines, extrapolate, interp, ax)
 
     # plot interpolated map
-    im = ax.imshow(Zi, cmap=cmap, vmin=vmin, vmax=vmax, origin='lower',
-                   aspect='equal', extent=extent,
-                   interpolation=image_interp)
+    if cnorm is None:
+        from matplotlib.colors import Normalize
```

Comment:
better to move this up with the other nested MPL imports to keep them together as much as possible
```suggestion
```